### DISCUSSION TITLE
AlpineJS seems to evaluate string value that it shouldn't

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3872] [7. Bugs (archived)] [answered]
For the following code (I use this string for the i8n plugin), I get the error `module.esm.js:534 Alpine Expression Error: Cannot set properties of undefined (setting 'description')` which is very strange, since AlpineJS should not do anything with that string. Furthermore, there is a translation for that string/identifier. So this error seems random, likely to be a bug.

![image](https://github.com/alpinejs/alpine/assets/1435963/6a004632-7373-4ba9-97c7-a19fee8ecc20)

**Comment by simonferndriger** [comment]
Since it breaks the app, I have to wrap it in a `try{}catch(){}` for now. This worked in an earlier version of AlpineJS if I am not mistaken. And I did not change this part of the code for a long time now.

**Reply by ekwoka** [reply, marked as answer]
This doesn't seem to be related to Alpine, but that `app.content.text` is undefined.

More context might help, but there is zero reason why Alpine should be doing anything with that string with what is provided here. Alpine doesn't touch your strings.

**Reply by simonferndriger** [reply]
You were right, sorry! I thought because the right part was underlined that the error was on the right part... 🙈
